core/server/master/src/main/java/alluxio/master/metastore/ReadOnlyInodeStore.java

```diff
@@ -29,9 +29,12 @@
  */
 public interface ReadOnlyInodeStore extends Closeable {
   /**
-   * @param id an inode id
-   * @param option the options
-   * @return the inode with the given id, if it exists
+   * Gets the inode if it exists.
+   *
+   * @param   id      the inode identifier
+   * @param   option  the {@link ReadOption} for the inode
+   * @return  the inode with the given id if it exists,
+   *          empty otherwise
```

Comment:
You specified the return in the `return`tag, but the description is very poor yet. Talk about the operations the function will do. It will check if the Inode exists and return it, if not will return a empty value.